app/experimenter/experiments/tests/test_models/test_models_nimbus.py

```diff
@@ -444,6 +444,28 @@ def test_can_review_true_for_non_requesting_user(self):
 
         self.assertTrue(experiment.can_review(UserFactory.create()))
 
+    def test_results_ready_true(self):
+        experiment = NimbusExperimentFactory.create()
+
+        NimbusChangeLogFactory.create(
+            experiment=experiment,
+            old_status=NimbusExperiment.Status.DRAFT,
+            new_status=NimbusExperiment.Status.LIVE,
+            changed_on=datetime.date(2019, 5, 1),
+        )
+        self.assertTrue(experiment.results_ready)
```

Comment:
@jodyheavener Here take a look at this, it would be nice if we could also incorporate providing like a start date for things that need to take place in teh past.....

Reply:
I did something similar in the legacy `create_with_status` where it picks a random day in teh past and does each lifecycle step 1 day later

https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/tests/factories/legacy.py#L144-L164